.github/workflows/release.yml

```diff
@@ -0,0 +1,48 @@
+name: Release
+on:
+  release:
+    types:
+      - created
```

Comment:
Should I introduce the release-please action?

Reply:
Yes please 🙂 If we can get this workflow in sync with release-please, that would be awesome